typescript/src/lib/bulk_command_execution/v0_bulk_command_api/v0_bulk_commands.ts

```diff
@@ -139,6 +150,16 @@ class CmdArgDeserializingVisitor implements V0CommandTypeVisitor {
         return ok(null);
     }
 
+    public visitGetServices(): Result<null, Error> {
+        this.deserializedCommandArgsPtr = undefined
```

Comment:
Same thing here - I'd make this to the protobuf Empty object so we don't get `undefined` or NPE